src/Image/ResizeConfiguration.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Image;
+
+/**
+ * Resize configuration used by the ResizeCalculator
+ *
+ * @author Martin Auswöger <martin@auswoeger.com>
+ */
+class ResizeConfiguration
+{
+    /**
+     * @var int
+     */
+    private $width = 0;
+
+    /**
+     * @var int
+     */
+    private $height = 0;
+
+    /**
+     * @var string
+     */
+    private $mode = self::MODE_CROP;
+
+    const MODE_CROP = 'crop';
```

Comment:
Class constants are usually at the very top of the class, no matter to what var they belong :)